src/main/java/frc/robot/util/ShuffleBoard/Field.java

```diff
@@ -25,12 +25,10 @@ public Field(Drivetrain drive){
     } 
     
     public void updateModulePositions(){
+        
         if (chassisPose != null){
             chassisPose = drive.getPose();
```

Comment:
To make sure this is never null, you should check the drivetrain's pose, not the previous pose from the last frame.